cmd/bootkube/start.go

```diff
@@ -40,6 +43,12 @@ func runCmdStart(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Invalid etcd etcdServer %q: %v", startOpts.etcdServer, err)
 	}
 
+	if startOpts.selfHostedEtcd {
+		if err := etcdutil.StartEtcd(); err != nil {
```

Comment:
Looks like our logging initialization comes after this (but glog is being used). Should either move this down, or move the logging init to be earlier.